### PR TITLE
5 DSC Schemes

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

**Comment by tool4ever** [comment]
wrong PR title?

**Comment by Fulgur14** [comment]
Not sure what happened there... I thought I uploaded those.

File: forge-gui/res/cardsfolder/upcoming/i_call_for_slaughter.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:I Call for Slaughter
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigToken | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, create three 1/1 red Devil creature tokens with "When this creature dies, it deals 1 damage to any target." They gain haste until end of turn. If a source you control would deal damage this turn, it deals that much damage plus 1 instead.
+SVar:TrigToken:DB$ Token | TokenAmount$ 3 | TokenScript$ r_1_1_devil_burn | PumpKeywords$ Haste | PumpDuration$ EOT | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | ReplacementEffects$ DamageEvent | SpellDescription$ If a source you control would deal damage this turn, it deals that much damage plus 1 instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/i_call_for_slaughter.txt:6]: Bad description

File: forge-gui/res/cardsfolder/upcoming/reality_is_mine_to_control.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Reality Is Mine to Control
+ManaCost:no cost
+Types:Ongoing Scheme
+T:Mode$ SpellCast | ValidCard$ Card | ValidActivatingPlayer$ You | Execute$ TrigAbandon | TriggerZones$ Command | TriggerDescription$ Whenever you cast a spell, you may abandon this scheme. If you do, copy that spell, and you may choose new targets for the copy.
+SVar:TrigAbandon:DB$ Abandon | Optional$ True | RememberAbandoned$ True | SubAbility$ DBCopy
+SVar:DBCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | MayChooseTarget$ True | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/reality_is_mine_to_control.txt:6]: Condition missing

File: forge-gui/res/cardsfolder/upcoming/you_exist_only_to_amuse.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:You Exist Only to Amuse
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigCharm | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, ABILITY
+SVar:TrigCharm:DB$ Charm | CharmNum$ Count$Compare X GE6.2.1 | Choices$ DBToken,DBAnimateAll | AdditionalDescription$ If you control six or more lands, choose both instead.
+SVar:DBToken:DB$ Token | TokenAmount$ 3 | TokenScript$ r_1_1_devil_burn | SpellDescription$ Create three 1/1 red Devil creature tokens with "When this creature dies, it deals 1 damage to any target."
+A:SP$ AnimateAll | ValidCards$ Creature.OppCtrl | Power$ 1 | Toughness$ 1 | RemoveAllAbilities$ True | Duration$ UntilYourNextTurn | SpellDescription$ Until your next turn, creatures your opponents control have base power and toughness 1/1 and lose all abilities.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/you_exist_only_to_amuse.txt:7]: Bad line

File: forge-gui/res/cardsfolder/upcoming/your_nightmares_are_delicious.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Your Nightmares Are Delicious
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigRepeatEach | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, each opponent who has more than five cards in hand discards cards equal to the difference. Then if fewer than three cards were discarded this way, you draw three cards.
+SVar:TrigRepeatEach:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBDiscard | SubAbility$ DBDraw
+SVar:DBDiscard:DB$ Discard | Defined$ Player.IsRemembered | NumCards$ X | Mode$ TgtChoose | RememberDiscarded$ True
+SVar:DBDraw:DB$ Draw | NumCards$ 3 | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ LT3 | SubAbility$ DBCleanUp
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/your_nightmares_are_delicious.txt:7]: Bad sub

File: forge-gui/res/cardsfolder/upcoming/your_nightmares_are_delicious.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Your Nightmares Are Delicious
+ManaCost:no cost
+Types:Scheme
+T:Mode$ SetInMotion | ValidCard$ Card.Self | Execute$ TrigRepeatEach | TriggerZones$ Command | TriggerDescription$ When you set this scheme in motion, each opponent who has more than five cards in hand discards cards equal to the difference. Then if fewer than three cards were discarded this way, you draw three cards.
+SVar:TrigRepeatEach:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBDiscard | SubAbility$ DBDraw
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/your_nightmares_are_delicious.txt:5]: Wrong repeat

**Review by tool4ever — APPROVED** [review]

**Pull request merged**